app/controllers/application/ApplicationController.scala

```diff
@@ -231,6 +231,13 @@ trait ApplicationController extends BackendController with SecureStorageControll
                 val pr: ProcessingReport = jsonValidator.validate(desJson, Constants.schemaPathApplicationSubmission)
                 if (pr.isSuccess) {
                   Logger.info("DES Json successfully validated")
+
+                  registrationDetails flatMap { _.applicantDetails } map { _.nino } match {
+                    case Some(leadExecutorNino) if nino!=leadExecutorNino =>
+                      Logger.error(s"[ApplicationController][submit] Submission attempt is NOT the lead executor")
+                    case _ =>
```

Comment:
This empty case seems to be redundant.

Reply:
It has been done on purpose for when the lead executor is equal to the logged in NINO + for the cases when an applicatant.nino is not present(which should never really happen)